EmailValidator/Validation/DNSCheckValidation.php

```diff
@@ -15,13 +15,13 @@ class DNSCheckValidation implements EmailValidation
     private $warnings = [];
 
     /**
-     * @var InvalidEmail
+     * @var InvalidEmail|null
      */
     private $error;
-    
+
     public function __construct()
     {
-        if (!extension_loaded('intl')) {
+        if (!function_exists('idn_to_ascii')) {
             throw new \LogicException(sprintf('The %s class requires the Intl extension.', __CLASS__));
```

Comment:
How does `idn_to_ascii` correlates to `intl` extension? Are they the same?
If so, please also update the error message.

Reply:
`symfony/polyfill-intl-idn` will take over when the extension isn't loaded, thus the condition needs to check for the function used.